### PR TITLE
Added some productivity tools

### PR DESCRIPTION
**Pull request by george-gca**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

- ConEmu - Windows console emulator with tabs
- Wave Terminal - open-source AI-native terminal
- Termux - terminal emulator app for Android
- Zed - high-performance, multiplayer code editor
- Lapce - powerful Code Editor written in Rust
- NVTOP - top like command for monitoring GPUs
- Rclone - rsync for cloud storage
- tldr - collaborative cheatsheets for console commands
- aria2 - multi-protocol, cross platform CLI download utility
- Bash-it - community Bash commands and scripts
- DVC - ML Experiments and Data Management with Git
- Starship - cross-shell, feature rich prompt made in rust
- Tilix - tiling terminal emulator for Linux using GTK+ 3
- mu-repo - help in dealing with multiple git repositories
- thefuck - corrects your previous console command
- bat - cat with syntax highlight and line numbers
- Delta - syntax-highlighting pager for git, diff, and grep
- dust - more intuitive du command in rust
- eza - modern, maintained replacement for ls
- tailspin - log file highlighter

### CODE REVIEW

File: collections/productivity-tools/index.md [hunk 1]
```diff
@@ -18,6 +18,26 @@ items:
...
+ - termux/termux-app
+ - zed-industries/zed
+ - lapce/lapce
+ - Syllo/nvtop
+ - rclone/rclone
+ - tldr-pages/tldr
+ - aria2/aria2
+ - Bash-it/bash-it
+ - iterative/dvc
+ - starship/starship
```
Comment (tomthorogood) [inline collections/productivity-tools/index.md:32]: 🚀 ❤️

**Review by tomthorogood — APPROVED** [review]
